src/project.ts

```diff
@@ -262,7 +273,7 @@ class Project extends ServiceObject {
        */
       setMetadata: {
         reqOpts: {
-          method: 'PUT',
+          method: 'PUT' as METHOD_TYPE,
```

Comment:
instead of duplicating the method list here, just type the `reqOpts` object as `Options`